fastify.d.ts

```diff
@@ -65,7 +73,8 @@ declare namespace fastify {
     Server extends http.Server,
     Logger extends FastifyBaseLogger = FastifyBaseLogger
   > = FastifyServerOptions<Server, Logger> & {
-    http?: http.ServerOptions | null
+    http?: http.ServerOptions | null,
+    childLoggerFactory?: FastifyChildLoggerFactory
```

Comment:
I believe it should be added to the `FastifyServerOptions` instead of `FastifyHttpOptions`

Reply:
You're right, thanks for the feedback!